plugin/kubernetes/object/pod.go

```diff
@@ -12,6 +12,7 @@ type Pod struct {
 	Name      string
 	Namespace string
 	Deleting  bool
+        Phase     string
```

Comment:
ah, this is *very* expensive to add because it increases memory. Can't we use `Deleting` or figure out something there?

Reply:
@miekg Based on my limited understanding of the codebase (I just reviewed it today for the first time), that's the easiest change. If memory is a concern, instead of storing the status itself, we could store a bool to indicate whether it's running or not. What do you think?

The field will be less useful but will use less memory.

Reply:
Btw, the status is about 5-6 characters long (not sure how much underlying memory this actually means though), is that really too much? 🙄

Reply:
Let me check if we can use Deleting.

Reply:
Can confirm, Deleting will be false for all the Pods, including the historical ones. We can't use Deleting here.